Ingredients/Database/CategoriesRepository.cs

```diff
@@ -46,9 +46,9 @@ public async Task<string> CreateCategory(Category category)
             async tx =>
             {
                 var result = await tx.RunAsync(
-                    "CREATE (c:Category $params) " +
-                    "RETURN id(c)",
-                    new { Object = parameters });
+     "CREATE (c:Category $Object) " +
+     "RETURN id(c)",
+     new { Object = parameters });
```

Comment:
This seems to be a chronic issue in this PR. Instead of rebasing on the corresponding branches you take the changes and copy them over (or cherry pick them) directly. Should the base branches be updated you have no direct way of updating.

Adding to this the original work put in by its authors is lost, its not in bad faith (I hope), but you are committing others work under your name.

Reply:
I didnt do it in any bad faith i jsut changed that myself on my branch because else i couldnt test if my adding categories via the csv file works as i rely on createcategory which did not work when i pulled from dev so i found the mistake and fixed it didnt know that u had arleady fixed it in the meantime